source/code-snippets/usage-examples/deleteOne.js

```diff
@@ -13,12 +13,12 @@ async function run() {
     const database = client.db("sample_mflix");
     const movies = database.collection("movies");
 
-    // Query for a movie that has a title of type string
-    const query = { title: { $type: "string" } };
+    // Query for a movie that has title "Annie Hall"
+    const query = { title: "Annie Hall" };
```

Comment:
Please update the typescript example as well

Reply:
The file is the same, no need to update

Reply:
Nevermind! You are right this doesn't introduce changes that would require a new TS file (forgot that this was one that only had one code snippet)